### ISSUE TITLE
toggle artwork permanently 

### ISSUE THREAD

**Issue by Chuck217** [ampache/ampache#1999]
I asked this in the Google Groups section and was redirected here.  I am trying to permanently toggle artwork OFF,  as my library consists of custom songs/files, which does not have artwork at all.   When I am logged in, there is a check box, below the FILTER section on the left hand side, that allows artwork to be toggled off.  Is it possible to add a place to turn this on/off in the config of ampache?

Google Groups post: https://groups.google.com/forum/#!topic/ampache/K17Ew2K3vlg
Thank you
Chuck 

# Issue Template

## Environment

* Ampache version:
* Web server + version:
* Server operating system:
* Client operating system:

* Client type:
  * Web
  * Ampache
  * SubSonic
  * DAAP
  * UPnP
  * WebDAV

**Comment by lachlan-00** [comment]
seems like all it needs is a config option for Art::is_enabled()

**Comment by lachlan-00** [comment]
I'm away for the weekend after tonight but this is what we'll do. If someone wants to be an Ampache contributor this is an easy one!

i think a config option inside ampache.cfg.php near the playlist_art section ("disable_art"). bump the config version and comment the new option
```
;disable_art = "true"
```

then update the art::is_enabled function in the art class
```
    public static function is_enabled()
    {
        if (self::$enabled && !AmpConfig::get('disable_art')) {
            return true;
        }

        return false;
```

**Comment by lachlan-00** [comment]
will look at moving this into #1957

**Comment by AshotN** [comment]
Was this done in #1957 ?

**Comment by lachlan-00** [comment]
No, not done. i want to move it out of the sidebar into the topbox
